user_guide_src/source/libraries/caching.rst

```diff
@@ -280,7 +280,7 @@ Drivers
 File-based Caching
 ==================
 
-Unlike caching from the Output Class, the driver file-based caching
+Unlike caching from the Output Class, the file-based caching 
```

Comment:
CI4 does not have the Output Class.
It seems it indicates "Web Page Caching" https://codeigniter4.github.io/CodeIgniter4/general/caching.html

The space in the end of the line is not needed.